### DISCUSSION TITLE
Helix not fetching grammar

### DISCUSSION THREAD

**Discussion by IsotoxalDev** [helix-editor/helix#8768] [Troubleshooting]
I am on the master branch of helix. and this is my languages.toml
![image](https://github.com/helix-editor/helix/assets/62714538/cf1e2891-98bc-40b2-b2c5-a46e2a93c03f)
`hx --grammar fetch` & `build` does nothing.

**Comment by pascalkuthe** [comment]
please use the issue template otherwire you r bugs are not actionable, show the output that hx --grammar fetch and build produce

**Comment by sicher** [comment]
I think you should have "[[grammar]]" in the file, not "[[grammer]]".
